Formula/red-tldr.rb

```diff
@@ -0,0 +1,17 @@
+class RedTldr < Formula
+  desc "Used to help red team staff quickly find the commands and key points"
+  homepage "https://payloads.online/red-tldr/"
+  url "https://github.com/Rvn0xsy/red-tldr/archive/refs/tags/v0.4.tar.gz"
+  sha256 "47894a194da21e676901985909ba60c616f8b832df7e74834ae3f2da44b64b02"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    system "red-tldr"
```

Comment:
Can we check some output here? Can we also test some functionality, e.g. `red-tldr search`?

Reply:
Thanks for your advice